services/alert/handlers.go

```diff
@@ -440,6 +439,21 @@ func newMatchHandler(match string, h alert.Handler, d HandlerDiagnostic) (*match
 }
 
 func (h *matchHandler) Handle(event alert.Event) {
+	defer func() {
+		if r := recover(); r != nil {
+			switch r := r.(type) {
+			case string:
+				h.diag.Error("recovered from panic", errors.New(r))
+			case error:
+				h.diag.Error("recovered from panic", r)
+			}
+		}
+	}()
+
+	if h.h == nil {
+		panic("h.h is nil")
+	}
```

Comment:
This looks like a debug check, do we want to leave it in?

Reply:
Doh, I thought I removed it.